src/app/components/TransactionListItem/TransactionListItem.tsx

```diff
@@ -4,14 +4,16 @@ import { Icon } from "app/components/Icon";
 import { Label } from "app/components/Label";
 import React from "react";
 
-type TransactionListItemProps = {
+export type TransactionListItemProps = {
```

Comment:
Any reason for this being exported?

Reply:
This is going to be required in `Notifications` component, [here](https://github.com/ArkEcosystem/desktop-wallet/commit/e1270220354f88e0c359bcfa26d927a5610afaf9#diff-d07d23621008e0910c251c32bca4d76cR22)

Is it better to add it in models.ts?

Reply:
Yep, that should go into the models file in that case.